migrations/sqls/20240430163831-add-marked-jobs-up.sql

```diff
@@ -0,0 +1,11 @@
+CREATE TYPE job_mark_type AS ENUM ('SAVE', 'NOT_INTERESTED');
+
+CREATE TABLE job_marks (
+  id                   uuid                            NOT NULL DEFAULT public.uuid_generate_v4() PRIMARY KEY,
+  identity_id          uuid                            NOT NULL,
+  job_id               uuid                            NOT NULL,
```

Comment:
don't use job_id or any job title for now we need to keep consistency on naming all jobs related name to projects